### PR TITLE
Update ids, naming, etc. to OCA standards (medical & medical_emr)

### PR DESCRIPTION
**Pull request by lasley**
Hi All,

I believe this is good to go. I did end up removing the emr_data module for the sake of time. Will swoop that up at some point later.

Hopefully I did better on the git side of things, too. Suggestions are welcome.

### CODE REVIEW

**Comment by lmignon** [comment]
@lasley I've some probs with this PR. This PR includes changes of https://github.com/OCA/vertical-medical/pull/27/files but the commit history is lost. You must rebase your branch on #27 without squashing the commits.

**Comment by lasley** [comment]
Hi @lmignon 

Sorry, I was trying to follow the OCA rule of squashing commits when necessary. I only touched my own commits, and the ones I squashed were related to each other or were just cases of work duplication on the branches.

Now that I'm looking at it though, I see the problem with squashes. Is there even a way for me to reverse this?

File: medical/models/medical_physician.py [hunk 1]
```diff
@@ -111,102 +107,3 @@ def create(self, cr, uid, vals, context=None):
         vals['groups_id'] = [(6, 0, group_ids)]
```
Comment (lasley) [inline medical/models/medical_physician.py]: Does anyone know the intent behind this `groups_ids`? It doesn't exist in the parent model `res.partner`, and isn't declared here.

I see it removed from `medical.patient`, just wanted to confirm there wasn't an awesome intent/idea behind this that should be preserved.

Comment (JonathanNEMRY) [inline medical/models/medical_physician.py]: Hello @lasley ,
It is no longer used since the patient became a res.partner instead of a res.users

Comment (lasley) [inline medical/models/medical_physician.py]: Ahhh thanks @JonathanNEMRY - I knew I recognized this variable name from somewhere. Will remove here

**Comment by lasley** [comment]
Closing in favor of #39

**Pull request closed without merging**